OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -151,7 +151,7 @@ public ConnectionFailedLogic(Widget widget, OrderManager orderManager, NetworkCo
 	public class ConnectionSwitchModLogic : ChromeLogic
 	{
 		[ObjectCreator.UseCtor]
-		public ConnectionSwitchModLogic(Widget widget, OrderManager orderManager, string password, NetworkConnection connection, Action onAbort, Action<string> onRetry)
+		public ConnectionSwitchModLogic(Widget widget, OrderManager orderManager, NetworkConnection connection, Action onAbort, Action<string> onRetry)
```

Comment:
Before we merge this PR, is this just fixing bitrot or an accidental rebase error?

Reply:
I put it in a separate commit now. These changes while they look simple often result in me reading through a lot of code in various places before I manage to find the right spot to insert to. This was an accidental find that was highlighted by @OmniSharp.